patches/@oclif+dev-cli+1.22.0.dev.patch

```diff
@@ -87,9 +87,19 @@ index 3e613e0..dd23903 100644
 +        await qq.emptyDir(ws);
 +        qq.cd(ws);
 +        const sources = [
-+            'bin', 'build', 'patches', 'typings', 'CHANGELOG.md', 'INSTALL.md',
-+            'LICENSE', 'package.json', 'npm-shrinkwrap.json', 'README.md',
-+            'TROUBLESHOOTING.md', 'automation/check-npm-version.js',
++            'CHANGELOG.md',
++            'INSTALL.md',
++            'LICENSE',
++            'README.md',
++            'TROUBLESHOOTING.md',
++            'automation/check-npm-version.js',
++            'bin',
++            'build',
++            'npm-shrinkwrap.json',
++            'package.json',
++            'patches',
++            'patches0',
++            'typings',
```

Comment:
The modifications to this patch file (`@oclif+dev-cli+1.22.0.patch`) are:
* Reformatted the `sources` array entries in alphabetical order, with one entry per line.
* Added folder `patches0` to the `sources` array entries. This is a new folder being added to the CLI repo in this PR. (See other comment.)
* Recreated the patch file in the usual way, in order to update the commit hashes and line numbers:
```
npx patch-package \@oclif/dev-cli
```
* Renamed the patch file to `.dev.patch` ([as required by the new version of patch-package](https://www.npmjs.com/package/patch-package#dev-only-patches), to avoid warnings/errors).